man/kernel-install.xml

```diff
@@ -164,6 +164,9 @@
 
     <para>If <varname>MACHINE_ID=</varname> is set and nonempty, it will be used as <replaceable>MACHINE-ID</replaceable>,
     overriding any automatic detection attempts.</para>
+
+    <para><varname>KERNEL_INSTALL_MACHINE_ID=</varname> is set for the plugins to the desired <replaceable>MACHINE-ID</replaceable>.</para>
+    <para><varname>KERNEL_INSTALL_BOOT_ROOT=</varname> is set for the plugins to the detected EFI system partition mount-point.</para>
```

Comment:
this is not correct. We actually allow boot entries to be either on the ESP or on a separate partition XBOOTLDR. see https://systemd.io/BOOT_LOADER_SPECIFICATION/. 

KERNEL_INSTALL_BOOT_ROOT actually points to where the kernels are dopped, which is either XBOOTLDR (if used) or the ESP otherwise. 

Maybe say

"it is set the path of the mount point of the partition where boot loader entries, kernel images and associated resources are placed" or so

Reply:
Sure, updated with similar wording